modules/ROOT/pages/migrate-mule3-apimanager.adoc

```diff
@@ -7,19 +7,19 @@ You migrate your existing Mule 3 applications that are managed by API Manager to
 
 . <<prepare-migration,Prepare for your migration>>
 +
-Based on how your existing Mule 3 application is configured, carefully review the prerequisites and determine how to plan the migration.
+Based on how your existing Mule 3 application is configured, carefully review the prerequisites to prepare for the migration.
```

Comment:
GLOBALs:

- The title for the topic set refers to "applications" but the subsequent text uses singular "application"; is this intentional?
- The narrative switches between "the migration" and "your migration"; does this matter?

Reply:
Done! Thanks!